desk/app/activity.hoon

```diff
@@ -277,6 +278,24 @@
       %adjust   (adjust +.action)
       %allow-notifications  (allow +.action)
     ==
+  ::
+      %egg-any
+    =+  !<(=egg-any:gall vase)
+    ?-  -.egg-any
+        ?(%15 %16)
+      ?.  ?=(%live +<.egg-any)
+        ~&  [dap.bowl %egg-any-not-live]
+        cor
+      =/  bak
+        (load -:!>(*versioned-state:load) +>.old-state.egg-any)
+      ::  restore volume settings, but keep any we've explicitly set ourselves,
+      ::  and restore activity summaries  ::REVIEW  for unintended side-effects
```

Comment:
they need these sources to exist in the index because if they have unreads here and try to read them before getting new events it will just crash. the least we can do here is grab all the new sources and bunt the index and place them in the indices. maybe even give them a floor:

```hoon
=.  indices
  %.  indices
  %~  uni  by
  %-  ~(urn by activity:bak)
  |=  [=source:a as=activity-summary:a]
  ?~  unread.as  *index:a
  index(floor.reads (sub time.u.unread.as (div ~s1 100)))
```

although I'm inclined to say even with this it might not be enough, may just be better to import indices same way you do activity.